<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>baza danych</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="surname">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="street">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>160</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="street_number">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="city">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="country">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="date">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>80</y>
      <width>76</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="phone_number">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>230</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>26</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Imię:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>9</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nazwisko:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>60</y>
      <width>82</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data urodzenia:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>140</y>
      <width>29</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ulica:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numer domu:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>39</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Miasto:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Państwo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>210</y>
      <width>87</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numer telefonu:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adres e-mail:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Miejsce zamieszkania:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dane kontaktowe:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zapisz">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>zapisz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wyczysc">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>wyczyść</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="namebox">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>150</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Imię</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="surnamebox">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>170</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nazwisko</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="miesiacbox">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>80</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>miesiąc</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dzienbox">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>60</y>
      <width>76</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dzień</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="phone_numberbox">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>190</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Numer telefonu</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rokbox">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>100</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>rok</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data urodzenia:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyszukiwanie:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="emailbox">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>210</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Adres e-mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data urodzenia:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="countrybox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>120</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Państwo</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="street_numberbox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>80</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Numer domu</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="citybox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>100</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Miasto</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="streetbox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>60</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ulica</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wyszukaj">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>250</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>wyszukaj</string>
    </property>
   </widget>
   <widget class="QLabel" name="wyniki_ostatniego">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>350</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Ok">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>470</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>dodaj</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>450</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj przykładowe rekordy:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>300</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>280</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyniki ostatniego wyszukiwania:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>411</width>
      <height>111</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;wybierz rekord do wyświetlenia&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>usuń</string>
    </property>
   </widget>
   <widget class="QLabel" name="rekordy">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ilość rekordów: 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>wczytaj z pliku</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>zapisz do pliku</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
